### ISSUE TITLE
#sel save <filename>

### ISSUE THREAD

**Issue by ThebesAndSound** [cabaletta/baritone#1116]
because this would have been useful for me today and its generally pretty cool

**Comment by ThebesAndSound** [comment]
This is to save a schematic

**Comment by TheK0tYaRa** [comment]
Your idea is so cool that i almost forgot my idea

**Comment by Deauthorized** [comment]
don't see why not tbh

**Comment by 0-x-2-2** [comment]
Schematica already has this 😳

**Comment by ThebesAndSound** [comment]
> Schematica already has this 😳

yes of course it does, but then you have to manually select the area again with schematica when in the case I am talking about you already have it selected with baritone, this suggestion makes it quicker to save it.

may i suggest you finally add baritone integrations to future instead of trying to make other projects as lazy as your team ❤️

**Comment by 0-x-2-2** [comment]
![image](https://user-images.githubusercontent.com/9968651/69012360-fc15cc80-0942-11ea-8af5-8416865a8849.png)

**Comment by majorkush** [comment]
I thought this was part of baritone already lmfao :P Only came on to see what it was doing wrong. Big Bump to this idea... Amazing working on the Project to far, still amazed.. hope all is well cheers.

**Comment by Aurumaker72** [comment]
bump

**Comment by scorbett123** [comment]
> bump

Sending pointless messages like that is just useless. It notifies a bunch of people, all of whom will just click "mark as read". It won't make it get done any quicker.

**Comment by NexitySecond** [comment]
bump i want this too

**Comment by AryonLegacy** [comment]
bump pepeLaugh

**Comment by sudofox** [comment]
tfw i find @ThebesAndSound github issue while trying to build a canal
![Screenshot from 2022-01-25 10-14-45](https://user-images.githubusercontent.com/3847441/151004093-dd88f3ec-ce9d-4531-9978-a577eb5363dc.png)

**Comment by sudofox** [comment]
i am now trying to make a faster wheat farm and being able to load the areas that i wanted to autoplant wheat in would be fantastic
